tests/js/server/resilience/moving-shards-cluster.js

```diff
@@ -35,11 +35,15 @@ const wait = require("internal").wait;
 const supervisionState = require("@arangodb/cluster").supervisionState;
 
 function getDBServers() {
-  var tmp = global.ArangoClusterInfo.getDBServers();
-  var servers = [];
-  for (var i = 0; i < tmp.length; ++i) {
-    servers[i] = tmp[i].serverId;
+  var tmp, servers = [];
```

Comment:
Let's remove this "fix" here and make a separate PR to fix the global variable `servers` in all branches.